### PR TITLE
Try out new modules preset

### PR DESCRIPTION
**Pull request by pascalduez**
See https://github.com/babel/preset-modules

Monitor https://github.com/babel/preset-modules/issues/12 see if we need to re-add those two plugins.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -42,12 +44,13 @@ module.exports = (context, options) => {
...
 
   let presets = [
-    ['@babel/preset-env', envOpts],
+    esmodules ? '@babel/preset-modules' : ['@babel/preset-env', envOpts],
     '@babel/preset-react',
     '@babel/preset-flow',
   ];
 
   let plugins = [
+    '@babel/plugin-syntax-dynamic-import',
```
Comment (ziir) [inline index.js]: This addition is necessary only with `@babel/preset-modules` since included with  `@babel/preset-env`, correct?

**Comment by ziir** [comment]
It would be interesting to add an async/await test case on master and take a look at the diff this PR brings :)

File: index.js (hunk 1 above)
Comment (pascalduez) [reply index.js]: Correct :)

**Comment by pascalduez** [comment]
> It would be interesting to add an async/await test case on master and take a look at the diff this PR brings :)

Exactly what I thought, done.

File: index.js (hunk 1 above)
Comment (ziir) [reply index.js]: Should we add it in this case only then?

**Comment by ziir** [comment]
:+1: Looks great to me :)
I think this can be tested on a non-critical app in production

File: index.js (hunk 1 above)
Comment (pascalduez) [reply index.js]: Yup, I first added it with a condition, then though it was not a big deal. Will update.

**Comment by pascalduez** [comment]
Closing in favor of #20

**Pull request closed without merging**